clog/loggers.py

```diff
@@ -320,7 +353,40 @@ def log_line(self, stream, line):
             )
             self.metrics.monk_exception()
 
-    def _log_line_no_size_limit(self, stream, line, can_flush_buffer=True):
+    def _get_schema(self, stream):
+        return {
+            "type": "record",
+            "namespace": "scribe_log",
+            "name": stream,
+            "doc": ("This schema is not used to actually serialize or deserialize messages,"
+                "but stream will be assigned to this schema."),
+            "fields": [
+                {"type": "string", "name": "log_line", "doc": "log line string"}
+            ]
+        }
+
+    def _register_schema(self, stream):
+        self._setup_schematizer_if_needed()
+        try:
+            self.schematizer_client.register_schema_from_schema_json(
+                namespace='scribe_log',
+                source=stream,
+                schema_json=self._get_schema(stream),
+                source_owner_email=str(config.owner_email),
+                contains_pii=False,
+                cluster_type='scribe',
```

Comment:
Why is the cluster_type needed here? Shouldn't it be automatically calculated from the stream_policy? In the near future we might want to send new streams to non scribe clusters.

Reply:
It uses existing version of API. The new API, where cluster_type should be determined by other features is not ready yet.